test/functional/test_framework/script_util.py

```diff
@@ -28,15 +29,18 @@
 # Output:      8 [Amount] + 1 [scriptPubKeyLen] = 9 bytes
 #
 # Hence, the scriptPubKey of the single output has to have a size of at
-# least 22 bytes, which corresponds to the size of a P2WPKH scriptPubKey.
-# The following script constant consists of a single push of 21 bytes of 'a':
-#   <PUSH_21> <21-bytes of 'a'>
-# resulting in a 22-byte size. It should be used whenever (small) fake
-# scriptPubKeys are needed, to guarantee that the minimum transaction size is
-# met.
-DUMMY_P2WPKH_SCRIPT = CScript([b'a' * 21])
-DUMMY_2_P2WPKH_SCRIPT = CScript([b'b' * 21])
-
+# least 5 bytes. DUMMY_P2WPKH_SCRIPT should be used when requiring
+# a spendable output, and DUMMY_MIN_OP_RETURN_SCRIPT when spendability
+# isn't required.
+MIN_STANDARD_TX_NONWITNESS_SIZE = 65
+MIN_PADDING = MIN_STANDARD_TX_NONWITNESS_SIZE - 10 - 41 - 9
+assert MIN_PADDING == 5
+
+# This script cannot be spent, side-stepping nValue standardness checks
+DUMMY_MIN_OP_RETURN_SCRIPT = CScript([OP_RETURN] * MIN_PADDING)
+
+# This can be spent, one shorter due to <push_MIN_PADDING> opcode
+DUMMY_MIN_SPEND_SCRIPT = CScript([b'a' * (MIN_PADDING - 1)])
```

Comment:
Wouldn't `[RETURN FALSE FALSE FALSE FALSE]` be a better min op_return script, since that way it passes standardness rules?

For a spendable (but necessarily non-standard) output, `[NOP NOP NOP NOP TRUE]` might be more explicit? But I don't think that case ever makes sense -- if it's a "dust" amount, that you don't care about, it's better to send it to fees and use a 0-value OP_RETURN output; if it's not a dust amount and you do care about it, you'll want to secure the funds and use something that requires an output to redeem. Saving a few bytes in the bip68 functional test doesn't seem worth the effort?

`assert len(*_MIN_*_SCRIPT) == MIN_PADDING` might be clearer than explaining all the off-by-one things?

Reply:
> Wouldn't [RETURN FALSE FALSE FALSE FALSE] be a better min op_return script, since that way it passes standardness rules?

done

> For a spendable

Leaving for another PR

> assert len(*_MIN_*_SCRIPT) == MIN_PADDING might be clearer than explaining all the off-by-one things?

Done